java/sage/media/sub/VobSubSubtitleHandler.java

```diff
@@ -46,24 +54,25 @@ public void loadSubtitlesFromFiles(sage.MediaFile sourceFile)
     try
     {
       subFile = getLoadableSubtitleFile(sourceFile, new java.io.File(subs[0].getPath()));
-      inStream = sage.IOUtils.openReaderDetectCharset(subFile, sage.Sage.BYTE_CHARSET);
+      inStream = sage.IOUtils.openReaderDetectCharset(subFile, sage.Sage.BYTE_CHARSET, sourceFile.isLocalFile());
       String line = inStream.readLine();
-      if (line != null && line.indexOf("VobSub index file") == -1)
+      if (line != null && line.contains("VobSub index file"))
       {
         if (sage.Sage.DBG) System.out.println("Invalid VobSub IDX file, bad comment on first line: " + line);
         return;
       }
       String orgSubPath = subs[0].getPath();
-      bitmapFile = getLoadableSubtitleFile(sourceFile, new java.io.File(orgSubPath.substring(0, orgSubPath.length() - 3) + "sub"));
-      if (bitmapFile == null)
+      bitmapFile = new java.io.File(orgSubPath.substring(0, orgSubPath.length() - 3) + "sub");
+      if (!sourceFile.isLocalFile())
       {
-        if (sage.Sage.DBG) System.out.println("Invalid VobSub; missing the .sub file for .idx file: " + orgSubPath);
-        return;
+        sage.NetworkClient.getSN().requestMediaServerAccess(bitmapFile, true);
```

Comment:
Once the file is opened, remove access only keeps it from being re-opened correct?